resources/monitoring/templates/kyma-additions/alertmanager.config.yaml

```diff
@@ -17,19 +17,19 @@ route:
     match:
       alertname: Watchdog
   {{- if and .Values.global.alertTools.credentials.victorOps }}
-  {{- if and (not (eq "" .Values.global.alertTools.credentials.victorOps.apikey)) (not (eq "" .Values.global.alertTools.credentials.victorOps.routingkey))}}
+  {{- if and (not (eq "" .Values.global.alertTools.credentials.victorOps.apikey)) (not (eq "" .Values.global.alertTools.credentials.victorOps.routingkey)) (not (eq "" .Values.global.alertTools.credentials.victorOps.matchExpression))}}
   - receiver: "victorOps"
     continue: true # If set to `false`, it stops after the first matching.
     match_re:
-      severity: critical
+      {{ tpl ( toYaml .Values.global.alertTools.credentials.victorOps.matchExpression | indent 4) . }}
   {{- end }}
   {{- end }}
   {{- if and .Values.global.alertTools.credentials.slack }}
-  {{- if and (not (eq "" .Values.global.alertTools.credentials.slack.channel)) (not (eq "" .Values.global.alertTools.credentials.slack.apiurl))}}
+  {{- if and (not (eq "" .Values.global.alertTools.credentials.slack.channel)) (not (eq "" .Values.global.alertTools.credentials.slack.apiurl))(not (eq "" .Values.global.alertTools.credentials.slack.matchExpression))}}
   - receiver: "slack"
     continue: true # If set to `false`, it stops after the first matching.
     match_re:
-      severity: critical
+      {{ tpl ( toYaml .Values.global.alertTools.credentials.slack.matchExpression | indent 4) . }}
```

Comment:
indention look like should be 6 instead of 4

Reply:
update it and many thanks.